### PR TITLE
chore: automate post-release

### PR DESCRIPTION
**Pull request by jrandolf-google**

### CODE REVIEW

File: .github/workflows/publish-on-tag.yml
Comment (OrKoN) [inline .github/workflows/publish-on-tag.yml]: I think GitHub Actions don't have write access to the repo?

Comment (mathiasbynens) [reply .github/workflows/publish-on-tag.yml]: They do. We might wanna configure the user name + email though. In case it’s useful, I used this for my old COVID-19 vaccination project: https://github.com/mathiasbynens/covid-19-vaccinations-germany/blob/4d8f58d7da0a3aedb3cb463454559d8745f2ede8/.github/workflows/update.yml#L34-L48

Comment (OrKoN) [reply .github/workflows/publish-on-tag.yml]: what about `permissions: read-all` at the top of the file? I thought it restricts the access? (although could not find a good doc about what permissions are meant by that)

Comment (OrKoN) [inline .github/workflows/publish-on-tag.yml]: also, it seems `git push` is missing?

File: .github/workflows/publish-on-tag.yml
Comment (OrKoN) [inline .github/workflows/publish-on-tag.yml:57]: the command sometimes says that it requires `npm run tsc`, should we re-compile after we applied the post version?

File: .github/workflows/publish-on-tag.yml
Comment (jrandolf) [reply .github/workflows/publish-on-tag.yml]: I'll remove the `read-all` permission. As for the username and email, I guess we can just use some trivial one.

Comment (OrKoN) [reply .github/workflows/publish-on-tag.yml]: Is it possible to only define write permissions that are needed for push?

File: .github/workflows/publish-on-tag.yml
Comment (mathiasbynens) [inline .github/workflows/publish-on-tag.yml]: In Bash, single-quoted strings don’t interpolate. I think this is what you want:

```suggestion
          git commit -m "chore: bump version to v${VERSION}"
```

Comment (jrandolf) [reply .github/workflows/publish-on-tag.yml]: Done!

File: .github/workflows/publish-on-tag.yml
Comment (mathiasbynens) [inline .github/workflows/publish-on-tag.yml]: Maybe

```suggestion
          git config user.name 'Puppeteer Bot'
```

File: .github/workflows/publish-on-tag.yml
Comment (jrandolf) [reply .github/workflows/publish-on-tag.yml]: Done.

**Review by mathiasbynens — CHANGES_REQUESTED** [review]

File: .github/workflows/publish-on-tag.yml
Comment (mathiasbynens) [inline .github/workflows/publish-on-tag.yml]: nit: missing LF at EOL

**Review by mathiasbynens — APPROVED** [review]

**Review by OrKoN — APPROVED** [review]
LGTM!

**Comment by OrKoN** [comment]
@jrandolf once this lands, do you want to cut a new release to test this out?

**Comment by jrandolf-google** [comment]
> @jrandolf once this lands, do you want to cut a new release to test this out?

Yes

File: .github/workflows/publish-on-tag.yml
Comment (OrKoN) [inline .github/workflows/publish-on-tag.yml]: suggestion: we could replace the command with `jq -r .version ./package.json` which should available by default in GitHub Actions

**Pull request merged**
